ompi/attribute/attribute.c

```diff
@@ -406,13 +411,24 @@ typedef struct attribute_value_t {
     int av_sequence;
 } attribute_value_t;
 
+/*
```

Comment:
Why is this necessary ? As far as I see there is still a single global object to handle the attributes ?

Reply:
Looks like it was done to use opal-object reference counting. The last one to tear down attributes calls the destructor. At some point Open MPI should really stop using globals and and this change is a step in the right direction.

Reply:
Im not arguing against avoiding globals, but  such a change deserve its own PR instead of being part of a very large one.

Reply:
No argument there. Up to @hppritcha to decide whether to move it to another PR.